src/App.js

```diff
@@ -1,26 +1,23 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Card from './Card/Card';
+import Header from './Header/Header';
+
```

Comment:
There is an approach in the React Style Guide to get rid of repeatable naming in the imports - by following component naming best practices:
https://github.com/airbnb/javascript/tree/master/react#naming
![image](https://user-images.githubusercontent.com/52723192/92835343-c8c99000-f3e3-11ea-84f6-785d9a6ba50d.png)
Please rename Card component and its css reference correspondingly.